modules/admediaBidAdapter.js

```diff
@@ -67,6 +69,16 @@ var AdmediaAdapter = function AdmediaAdapter() {
 
       adloader.loadScript(endpoint);
     }
+
+    if (userSync.iframeEnabled) {
+      userSync.registerSync('iframe', 'admedia', '//b.admedia.com/sync/iframe/');
```

Comment:
`userSync.iframeEnabled` does not exist.  You'd have to look at the config directly, like `config.getConfig('userSync');`

Reply:
Updated to use config.